samples/nrf9160/https_client/src/main.c

```diff
@@ -167,20 +168,25 @@ void main(void)
 		printk("Failed to connect to the LTE network, err %d\n", err);
 		return;
 	}
-	printk("OK\n");
 
-	err = getaddrinfo(HTTPS_HOSTNAME, NULL, &hints, &res);
+	snprintf(portstr, 6, "%d", HTTPS_PORT);
+
+	printk("Calling getaddrinfo() with connect addr %s port %s\n", HTTPS_HOSTNAME, portstr);
```

Comment:
nit: It's fine to have some extra logs. I would personally log the operation rather than the API call in a sample, e.g.
```
Looking up www.example.com
Resolved 93.184.216.34 (AF_INET)
```

Reply:
I would also omit printing the port in these logs about hostname lookup, and put that in the line logging `Connecting to .. ` instead. For hostname lookup, the port doesn't really matter, it's just copied in the output structure so that it can be passed to `connect()`.

Reply:
Perhaps something along this?
```
Waiting for network..
Looking up example.com
Resolved 93.184.216.34 (AF_INET)
Connecting to example.com:443
```